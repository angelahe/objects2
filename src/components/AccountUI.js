import React from "react";
import account from "./account";

class AccountUI extends React.Component {

    constructor () {
        super()
        this.state = {
            balance: 0
        }
    }

    render() {

        return(
            <div>
                <h1>Hello world from Account</h1>
                <p>My Account</p>

                <input id="startBalance" type="number" placeholder="Enter starting balance"/>
                <button>Create Account</button>

            </div>

        )
    }
}

export default AccountUI;