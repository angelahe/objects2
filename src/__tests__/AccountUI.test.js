import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import AccountUI from "../components/AccountUI";
import renderer from "react-test-renderer";

it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<AccountUI />, div);
    ReactDOM.unmountComponentAtNode(div);
});

it('renders without crashing', () => {
    shallow(<AccountUI />);
});

test('MathComp renders correctly', () => {
    const tree = renderer.create(<AccountUI />).toJSON();
    expect(tree).toMatchSnapshot();
});