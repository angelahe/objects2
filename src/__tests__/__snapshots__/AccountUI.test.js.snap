// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MathComp renders correctly 1`] = `
<div>
  <h1>
    Hello world from Account
  </h1>
  <p>
    My Account
  </p>
  <input
    id="startBalance"
    placeholder="Enter starting balance"
    type="number"
  />
  <button>
    Create Account
  </button>
</div>
`;
